proto/substrait/algebra.proto

```diff
@@ -41,6 +41,8 @@ message RelCommon {
     // Name (alias) for this relation. Can be used for e.g. qualifying the relation (see e.g.
     // Spark's SubqueryAlias), or debugging.
     string alias = 3;
+    // Sets (or resets) the output field names for any relation.
```

Comment:
Could we make it clear if the names here are alternative names or a single dotted name?

Reply:
Sure, although I'm not sure what you mean by 'single dotted name'.  Would the following comment explain it better?

```
// Assigns alternative output field names for any relation.  For example, if the relation outputs
// three fields (columns) then their names can be changed by assigning each new name to an 
// instance of output_names, in the same order in which they are emitted.
```

Reply:
Oh, so this is not the name of the subquery but its outputs.  Then it should work exactly like root names (so you can name struct subfields as well).  A reference to relroot's names field should be enough.

Reply:
// Assigns alternative output field names for any relation.  Equivalent to the names field
// in RelRoot but applies to the output of the relation this RelCommon is attached to.